src/genesis_proof.ipynb

```diff
@@ -44,7 +44,15 @@
     "We will now walk through the code that does all that (the code was written and run on a linux machine, running it on other operating systems might require adaptations).\n",
     "\n",
     "If you intend to run the code locally, you could use the following command to make sure you have all the required libraries:\n",
-    "> pip install numpy,pandas,plyvel,protobuf==3.20.0,tqdm,notebook,markupsafe==2.0.1"
+    "> pip install numpy,pandas,plyvel,protobuf==3.20.0,tqdm,notebook,markupsafe==2.0.1\n",
```

Comment:
While you are at it, can you fix this line as well and switch the commas with a space? same goes for the project readme